src/java/htsjdk/variant/variantcontext/CommonInfo.java

```diff
@@ -151,7 +173,8 @@ public boolean hasLog10PError() {
      */
     public double getPhredScaledQual() { return (getLog10PError() * -10) + 0.0; }
 
-    public void setLog10PError(double log10PError) {
+    /** set the Phred scaled quality score */
+    public void setLog10PError(final double log10PError) {
```

Comment:
The comment for this method is incorrect -- `log10PError` is not Phred-scaled, as @ldgauthier suspected. You can see this by looking at the implementation of the `getPhredScaledQual()` method above.